HTML/Default/html/LiveSearch.js

```diff
@@ -60,6 +60,7 @@ LiveSearch = {
 					links : {
 						track: new Ext.Template( webroot + 'songinfo.html?player={player}&item={id}'),
 						album: new Ext.Template( webroot + 'clixmlbrowser/clicmd=browselibrary+items&mode=albums&linktitle=' + SqueezeJS.string('album') + '%20({title})&album_id={id}&player={player}/index.html?index=0'),
+						work: new Ext.Template( webroot + 'clixmlbrowser/clicmd=browselibrary+items&mode=works&linktitle=' + SqueezeJS.string('works') + '%20({title})&work_id={id}&from_search={terms}&player={player}/index.html?index=0'),
```

Comment:
Looks like I anticipated this question ;) - see the comment in Queries.pm:

```
# If we've got here from a search, we don't want to show the album unless it matches all the user's search criteria.
# This matters for a Works search: we've shown the user a Work because it matches their criteria, but it is possible
# that not all albums containing the Work match the all the criteria. (In this context, a work is a group of albums.)
```

Reply:
But the question remains: wouldn't we need it on all links?

Reply:
Because for Album, Artist and Song results, each displayed search result represents a single row from albums, contributors or tracks and when clicked we run the appropriate query - album, artist or track - having already determined the primary key.

For Work, it's different, because each displayed result represents a group of albums*, not all of which would necessarily match all search criteria.

* more correctly, a work is a group of album subsets, ie it may represent only some of the tracks on each album.